bouncer.go

```diff
@@ -556,15 +607,9 @@ func handleStreamCache(bouncer *Bouncer) error {
 	return nil
 }
 
-func crowdsecQuery(bouncer *Bouncer, stringURL string, isPost bool) ([]byte, error) {
+func crowdsecQuery(bouncer *Bouncer, stringURL string, isPost bool, data []byte, expectedStatusCodes []int) ([]byte, error) {
```

Comment:
maybe you can replace isPost by the presence of data ?

Reply:
Done.